<!--%Tag(FULL)%-->
<launch>
	
	<param name="music_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/dir/music"/>
	<param name="bextract_args" value="-mfcc" />
	<param name="feature_subs" value="1" />
	
	<node pkg="audiogesture" type="audio_gesture_server" name="AudioGestureServer" output="screen"/>
	
	<node pkg="audiogesture" type="music_extractor" name="MusicFeatureExtractor" output="screen"/>
	
	<node pkg="audiogesture" type="music_extractor_buffer" name="MusicFeatureExtractorBuffer" output="screen"/>
	
	<node pkg="audiogesture" type="sample_listener" name="SampleListener" output="screen"/>
	
	<node pkg="audiogesture" type="collection_generator" name="CollectionGenerator" output="screen"/>
	
	<node pkg="audiogesture" type="feature_normalizer" name="FeatureNormalizer" output="screen"/>
	
	<node pkg="audiogesture" type="gesture_receiver" name="GestureReceiver" output="screen"/>
	
	<node pkg="audiogesture" type="gesture_publisher" name="GesturePublisher" output="screen"/>
	
	<node pkg="audiogesture" type="sample_player" name="SamplePlayer" output="screen"/>
	
</launch>
<!--%EndTag(FULL)%-->



<!--
	
  <node pkg="audiogesture" type="audio_gesture_trainer" name="AudioGestureTrainer" output="screen"/>

-->

